src/rest_framework_dso/permissions.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import permissions
+from dso_api.datasets import models
+
+
+def fetch_scopes_for_model(model):
+    """ Get the scopes for a Django model, based on the Amsterdam schema information """
```

Comment:
Kunnen we de fetch_scopes_for_model niet op een of andere manier cachen. Nu wordt voor ieder request een apart DB request gedaan om scopes op te halen

Reply:
Dat had ik idd ook al overwogen. Ik had overwogen er de functools.lrucache op zetten, maar die wordt niet automatisch geinvalideerd. Ik ga kijken of ik er een cache met timeout op kan zetten.

Reply:
Ik vond een mooie expiring memoize cache (cachetools). Deze is toegevoegd. TTL nu op 60 minuten.